spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -380,6 +380,16 @@ def method_missing(sym, *args)
       end
     end
 
+    context "when with instance_miq_request_new" do
+      it "and @lastaction = show, @display = instances" do
```

Comment:
I don't really like this `it`, `"and @lastaction = show, @display = instances"` doesn't say
much here really.

@skateman What do you think? What would you suggest here?